core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/chain/BackupDatalakeDatabaseFlowEventChainFactory.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.core.flow2.chain;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.salt.update.SaltUpdateEvent;
+import com.sequenceiq.cloudbreak.core.flow2.event.DatabaseBackupTriggerEvent;
+import com.sequenceiq.cloudbreak.reactor.api.event.StackEvent;
+import com.sequenceiq.flow.core.chain.FlowEventChainFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.Queue;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+import static com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.dr.backup.DatabaseBackupEvent.DATABASE_BACKUP_EVENT;
+
+@Component
+public class BackupDatalakeDatabaseFlowEventChainFactory implements FlowEventChainFactory<DatabaseBackupTriggerEvent> {
+    @Override
+    public String initEvent() {
+        return FlowChainTriggers.DATALAKE_DATABASE_BACKUP_CHAIN_TRIGGER_EVENT;
+    }
+
+    @Override
+    public Queue<Selectable> createFlowTriggerEventQueue(DatabaseBackupTriggerEvent event) {
+        Queue<Selectable> chain = new ConcurrentLinkedQueue<>();
+        String selector = DATABASE_BACKUP_EVENT.event();
```

Comment:
Nit: What is the benefit of having an explicit variable instead of just inlining the value in line 27 below?

Reply:
There is no benefit. I can make a change inlining the value.

Reply:
addressed.